error.js

```diff
@@ -0,0 +1,21 @@
+exports.handlePsqErrors = (err, req, res, next) =>{
+    if(err.code === "22P02" || err.code === "23502") {
```

Comment:
Currently nothing your server does could cause these errors. You're just doing a select all from a table. Please don't add code outside the scope of the ticket.